packages/cxl-ui/src/index.js

```diff
@@ -2,21 +2,19 @@ import '@vaadin/vaadin-notification';
 import '@vaadin/vaadin-progress-bar';
 import * as Headroom from 'headroom.js';
 
+export { CXLAccordionCardElement } from './components/cxl-accordion-card.js';
 export { CXLAppLayoutElement } from './components/cxl-app-layout.js';
 export { CXLCardElement } from './components/cxl-card.js';
+export { CXLLikeOrDislikeElement } from './components/cxl-like-or-dislike.js';
 export { CXLMarketingNavElement } from './components/cxl-marketing-nav.js';
+export { CXLPaywallElement } from './components/cxl-paywall.js';
+export { CXLPlaybookAccordionElement } from './components/cxl-playbook-accordion.js';
+export { CXLPlaybookProgressBarElement } from './components/cxl-playbook-progress-bar.js';
+export { CXLSaveFavoriteElement } from './components/cxl-save-favorite.js';
 export { CXLSectionElement } from './components/cxl-section.js';
+export { CXLStarRatingElement } from './components/cxl-star-rating.js';
 export { CXLTabsSliderElement } from './components/cxl-tabs-slider.js';
-
 // Order matters.
 export { CXLVaadinAccordionElement } from './components/cxl-vaadin-accordion.js';
```

Comment:
Are we sure we're not creating a problem here?

I remember this import order possibly affects how style modules are loaded, which may or may not be a thing for newest vaadin-themeable-mixin, but that's yet to be merged.

Reply:
I'll see if there's a way for us to omit certain files.